### PR TITLE
Fix pom dependencies

### PR DESCRIPTION
**Pull request by tnovo**
Addresses issues on #49

### CODE REVIEW

**Comment by pedrorijo91** [comment]
we probably also need to update the providers

* https://github.com/feedzai/feedzai-openml-python
* https://github.com/feedzai/feedzai-openml-r
* https://github.com/feedzai/feedzai-openml-java
* internal (private) Feedzai provider

will try to have a look during this week

File: openml-utils/pom.xml [hunk 1]
```diff
@@ -44,13 +44,31 @@
         </dependency>
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
-            <artifactId>jackson-annotations</artifactId>
```
Comment (pedrorijo91) [inline openml-utils/pom.xml:47]: just noticed, why was this removed?

Comment (tnovo) [reply openml-utils/pom.xml:47]: It's not needed directly.
